inventory/views.py

```diff
@@ -66,9 +66,31 @@ def get_context_data(self, *args, **kwargs):
         context = super(CatalogueMixin, self).get_context_data(*args, **kwargs)
         context.update({
             'item_type': self.item_type,
+            'tabs': self.get_tabs()
         })
         return context
 
+    def get_tabs(self):
+        """
+        Returns a list of dictionary objects for tab information
+        :return:
+        """
+        tabs = []
+        for item_type in Item.get_item_contenttypes():
+            if self.request.user.has_perm(f'{item_type.app_label}.view_{item_type.model}'):
+                model_class = item_type.model_class()
+                tabs.append({
+                    'verbose': model_class.__name__,
+                    'icon_class': model_class.icon_class,
+                    'url': reverse("inventory:catalogue", kwargs={'type_id': item_type}),
+                    'selected': item_type == self.item_type,
+                })
+        return tabs
+
+
+class CatalogueInstructionsView(MembershipRequiredMixin, CatalogueMixin, TemplateView):
```

Comment:
Ideally this should be limited to those with `boardgames.view_boardgames` (and friends) but this'll work for now

Reply:
True, but didn't want to make the hassle. However, thinking about it now adding it in a later PR may be even more of a (if not forgotton) hassle